content/use-cases/contribute-code-to-rippled.md

```diff
@@ -0,0 +1,56 @@
+# Contribute Code to rippled
+
+Want to contribute code or a bug report to help improve rippled, the core peer-to-peer server that manages the XRP Ledger? Here’s a roadmap to the high-level tasks that’ll have you reviewing code and functionality in no time.
+
+{% set n = cycler(* range(1,99)) %}
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## <a href="https://github.com/ripple/rippled" target="_blank">Access the `rippled` repo</a> <i class="fa fa-external-link" aria-hidden="true"></i><!--#{ fix md highlighting_ #}-->
+
+`rippled` is an open-source project. You can take a look at `rippled` code simply by accessing the `rippled` GitHub repo. Before contributing or reporting bugs, we recommend that you get to know the code and developer experience by performing the following tasks.
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## [Set up and run a `rippled` server](xxxxx.html)
+
+Set up and run a `rippled` server to understand the developer experience and functionality of the core peer-to-peer server that manages the XRP Ledger. Anyone can run their own `rippled` server that follows the network and keeps a complete copy of the XRP Ledger.
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## [Try out XRP Ledger integration tools](xxxxx.html)
+
+Take a look at the various tools provided to help developers integrate with the XRP Ledger. From WebSocket and JSON-RPC API endpoints to the `ripple-lib` JavaScript library, take a look at the modes of integration offered to the developer community.
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## [Get a test XRP Ledger account and test XRP](xxxxx.html)
+
+Use the XRP Ledger Test Net to get a test account and test XRP. Connect your `rippled server` to the Test Net to make test calls and get to know the XRP Ledger.
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## [Set up your developer environment](xxxxx.html)
+
+<!-- TODO: awaiting links to a few rippled repo md files -->
+
```

Comment:
Suggestion: Add an extra step of "familiarize yourself with [`rippled`'s coding style](https://github.com/ripple/rippled/blob/develop/docs/CodingStyle.md).